<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Accueil</class>
 <widget class="QMainWindow" name="Accueil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>861</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Connexion">
     <attribute name="title">
      <string>Connexion</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>881</width>
        <height>371</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../images/Utc.qrc">:/images/utc.jpg</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="inscription">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>190</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Inscription</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_mdp">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>150</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_login">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_mdp">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>150</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot de passe :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_login">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>120</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Login :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connexion">
      <property name="geometry">
       <rect>
        <x>574</x>
        <y>190</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Connexion</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_erreur">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>230</y>
        <width>281</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;UT Profiler&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>inscription</zorder>
     <zorder>input_mdp</zorder>
     <zorder>input_login</zorder>
     <zorder>label_mdp</zorder>
     <zorder>label_login</zorder>
     <zorder>connexion</zorder>
     <zorder>label</zorder>
     <zorder>label_erreur</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>863</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../images/Utc.qrc"/>
 </resources>
 <connections/>
</ui>
